### PR TITLE
Removing BOOST_FOREACH usage.

### PR DESCRIPTION
**Pull request by timothy-king**

### CODE REVIEW

File: src/options/didyoumean_test.cpp [hunk 1]
```diff
@@ -426,340 +426,339 @@ set<string> getDebugTags()
...
+      "no-include-file",
+      "enable-idl-rewrite-equalities",
+      "disable-idl-rewrite-equalities",
+      "sets-propagate",
+      "no-sets-propagate",
+      "sets-eager-lemmas",
+      "no-sets-eager-lemmas",
+      NULL,
+  }; /* cmdlineOptions */
   int i = 0;
```
Comment (aniemetz) [inline src/options/didyoumean_test.cpp]: Is this intentional? Isn't this an indent of 3 chars?

Comment (timothy-king) [inline src/options/didyoumean_test.cpp]: M-x clang-format-buffer is what happened. Preference for what to do: revert these changes or keep?

Comment (aniemetz) [inline src/options/didyoumean_test.cpp]: I actually thinks it's better :-) Maybe we should discuss what we could use for introducing consistent formatting (automated) at one point.
